Makefile

```diff
@@ -57,6 +58,11 @@ proto-lint:
 	@$(DOCKER_BUF) lint --error-format=json
 .PHONY: proto-lint
 
+## proto-check-breaking checks if their is any breaking change to the protos
```

Comment:
Currently `make help` results in:

```
$ make help
 Choose a command run in celestia-app:
  help                                                                        Get more info on make commands.
  build                                                                       Build the celestia-appd binary into the ./build directory.
  install                                                                     Build and install the celestia-appd binary into the $GOPATH/bin directory.
  mod                                                                         Update go.mod.
  mod-verify                                                                  Verify dependencies have expected content.
  proto-gen                                                                   Generate protobuf files. Requires docker.
  proto-lint                                                                  Lint protobuf files. Requires docker.
  proto-check-breaking checks if their is any breaking change to the protos
```

but if we add a colon, this prints nicer

```suggestion
## proto-check-breaking: Check if there are any breaking change to protobuf definitions.
```

```
$ make help
 Choose a command run in celestia-app:
  help                   Get more info on make commands.
  build                  Build the celestia-appd binary into the ./build directory.
  install                Build and install the celestia-appd binary into the $GOPATH/bin directory.
  mod                    Update go.mod.
  mod-verify             Verify dependencies have expected content.
  proto-gen              Generate protobuf files. Requires docker.
  proto-lint             Lint protobuf files. Requires docker.
  proto-check-breaking   Check if there are any breaking change to protobuf definitions.
```

Reply:
Ah I didn't know about this - thanks